build_locally.sh

```diff
@@ -45,8 +45,8 @@ if ! have_noinstall "$@"; then
 fi
 
 # Test website using same script as Travis
-if have_travis "$@"; then
-  ./.travis.sh
+if have_check_arg "$@"; then
+  NOKOGIRI_USE_SYSTEM_LIBRARIES=true ./.check_build.sh
```

Comment:
One more nitpick: Declaring `NOKOGIRI_USE_SYSTEM_LIBRARIES=true` here again is code duplication.
Thus, please move this into the check_build.sh script using `export ...`

Otherwise looks good to me.

Reply:
Sure thing thanks a lot for taking your time to review!